packages/nameguard-react/src/components/Report/LoadingSkeleton.tsx

```diff
@@ -26,11 +26,6 @@ export const LoadingSkeleton = ({ parsedName }: LoadingSkeletonProps) => {
               <h1 className="pt-1 text-2xl md:text-4xl text-black font-semibold md:font-bold overflow-hidden overflow-ellipsis whitespace-nowrap ens-webfont">
                 {name}
               </h1>
-              <ReportFormattedDisplayName
```

Comment:
Why have we removed this? We want the user to instantly see results as they type (during the LoadingSkeleton) without having to wait for a server response to display back to the user the name that is being inspected.